tldw_Server_API/app/api/v1/endpoints/admin/admin_byok.py

```
@@ -0,0 +1,107 @@
+
+router = APIRouter()
+
+
+@router.get(
+    "/keys/users/{user_id}",
+    response_model=AdminUserKeysResponse,
+    dependencies=[Depends(require_roles("admin")), Depends(check_rate_limit)],
+)
+async def admin_list_user_byok_keys(
+    user_id: int,
+    principal: AuthPrincipal = Depends(get_auth_principal),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)